scintilla/include/Scintilla.h

```diff
@@ -49,6 +49,7 @@ typedef sptr_t (*SciFnDirect)(sptr_t ptr, unsigned int iMessage, uptr_t wParam,
 #define SCI_CHANGEINSERTION 2672
 #define SCI_CLEARALL 2004
 #define SCI_DELETERANGE 2645
+#define SCI_DELETECHAR 2719
```

Comment:
Anything in the scintilla directory comes from upstream, it will be overwritten next time its updated from upstream, so you need to get the changes into upstream.

Reply:
Ok. I didn't know and in first place it was not obvious to me